bpf/bpf_lxc.c

```diff
@@ -209,17 +210,22 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 			   orig_dip.p4, *dstID);
 	}
 
-	/* If the packet is in the establishing direction and it's destined
-	 * within the cluster, it must match policy or be dropped. If it's
-	 * bound for the host/outside, perform the CIDR policy check. */
-	verdict = policy_can_egress6(skb, tuple, *dstID,
-				     ipv6_ct_tuple_get_daddr(tuple));
+	if (!(cfg->flags & EP_F_SKIP_POLICY_EGRESS)) {
+		/* If the packet is in the establishing direction and it's destined
+		 * within the cluster, it must match policy or be dropped. If it's
+		 * bound for the host/outside, perform the CIDR policy check. */
+		verdict = policy_can_egress6(skb, tuple, *dstID,
+				ipv6_ct_tuple_get_daddr(tuple));
+	}
+	else {
```

Comment:
nit: With braces, the line should be `} else {`. With single-line contents inside each condition, the datapath code usually omits the braces entirely.

Reply:
I'm moving the check for IPv6 egress for consistency with IPv4 egress not having this. Will be fixed in next upload.